### PR TITLE
Fix button on dive log form getting stuck behind keyboard

### PR DESCRIPTION
**Pull request by mjmayank**
image

### CODE REVIEW

File: src/Screens/DiveLogsForms/index.tsx [hunk 1]
```diff
@@ -259,112 +260,117 @@ const SimpleDiveLogsForms: FunctionComponent<
...
-              style={[
-                styles.scrollContainer,
-                page !== stages.length && {
-                  marginBottom: isBelowHeightThreshold ? 100 : 80,
-                },
-              ]}>
-              <View style={styles.headerContainer}>
-                {page > 0 && page !== stages.length ? (
-                  <TouchableWithoutFeedback onPress={previous}>
+          <KeyboardAvoidingView
```
Comment (mjmayank) [inline src/Screens/DiveLogsForms/index.tsx:263]: most of the changes in this file are just whitespace changes. you can view the diff while ignoring whitespace changes like this: https://github.com/Zentacle/snorkel-app/pull/48/files?diff=split&w=1

**Comment by mjmayank** [comment]
Probably easiest to review this PR commit by commit since theyre all unrelated changes

**Comment by Nnanyielugo** [comment]
> Probably easiest to review this PR commit by commit since theyre all unrelated changes

Most of the significant ones are amplitude event changes. Apart from the form ones, the other two non amplitude related changes are pretty transparent changes.

**Comment by mjmayank** [comment]
Yeah, just meant to say that originally this PR was for the button so the title is wrong now

**Comment by Nnanyielugo** [comment]
> Yeah, just meant to say that originally this PR was for the button so the title is wrong now

That's fine. Just finished testing. You did fine work. Well done!

**Review by Nnanyielugo — APPROVED** [review]

**Comment by mjmayank** [comment]
could we push a build with these changes?

**Comment by Nnanyielugo** [comment]
> could we push a build with these changes?

Sure. On it.

**Pull request merged**